### PR TITLE
refactor(documentation-website): extract FAQ element in its own component

### LINKED ISSUE #523: Extract FAQ Element in it's own component
## Is your feature request related to a problem? Please describe

Currently we are creating similar elements in a loop, making the handling of state harder than necessary.

## Describe the solution you'd like

Extract the state handling, title and answer of a FAQ-Item into the component `faq-item.tsx`.

### PR DESCRIPTION
**Pull request by Hbrehman**
# The Pull Request is ready

- [x] fixes #523 and #505
- [x] all actions are passing
- [x] only fixes a single issue

## Overview

Extract FAQ element in new React component and fix up and down arrows on FAQ accordion.

## Documentation-Website

- [x] mobile view is usable
- [x] desktop view is usable
- [x] no a-tags are used directly (NavLink, MailLink, ExternalLink instead)
- [x] all new texts are added to the translation files (at least the english one)
- [x] tests have been added (if required)
- [x] shared code has been extracted in a different file

### CODE REVIEW

File: documentation-website/src/components/faq-item.tsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+                </h5>
+              </div>
+              <div>
+                <span>
+                  {isOpen ? <i>&#x23F6;</i> : <i>&#x23F7;</i>}
+                </span>
+              </div>
+            </div>
+          </div>
+          <div >
```
Comment (Idrinth) [inline documentation-website/src/components/faq-item.tsx:42]: this div is likely not necessary

File: documentation-website/src/components/faq-item.tsx [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+                    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+                    // @ts-expect-error
+                    <Lang lnkey={`home.faq.questions.title_${ index+ONE }`} />
+                  }
+                </h5>
+              </div>
+              <div>
+                <span>
+                  {isOpen ? <i>&#x23F6;</i> : <i>&#x23F7;</i>}
+                </span>
```
Comment (Idrinth) [inline documentation-website/src/components/faq-item.tsx:38]: do we need a span, an i and a div for a single element?

**Review by Idrinth — CHANGES_REQUESTED** [review]
Please also run `npm run lint-fix`, your formatting looks off.

File: documentation-website/src/components/faq-item.tsx (hunk 1 above)
Comment (Hbrehman) [reply documentation-website/src/components/faq-item.tsx]: I have removed unnecessary div.

File: documentation-website/src/components/faq-item.tsx (hunk 2 above)
Comment (Hbrehman) [reply documentation-website/src/components/faq-item.tsx]: removed unnecessary elements.

**Review by Idrinth — DISMISSED** [review]

**Comment by Idrinth** [comment]
> ## [Quality Gate Passed

those issues can be ignored for now

**Pull request merged**